pkgs/kubeshark/kubeshark/registry.yaml

```diff
@@ -0,0 +1,21 @@
+packages:
+  - type: github_release
+    repo_owner: kubeshark
+    repo_name: kubeshark
+    asset: kubeshark_{{.OS}}_{{.Arch}}
+    description: The API traffic viewer for Kubernetes providing real-time, protocol-aware visibility into Kubernetes internal network capturing, dissecting and monitoring all traffic and payloads going in, out and across containers, pods, nodes and clusters.
+    supported_envs:
+      - windows/amd64
+      - darwin
+      - linux
+    overrides:
+      - goos: windows
+        goarch: amd64
+        asset: kubeshark.exe
+    checksum:
+      type: github_release
+      asset: kubeshark_{{.OS}}_{{.Arch}}.sha256
+      file_format: regexp
```

Comment:
`file_format: raw` is better here since the checksum file only contains the sha

Reply:
This checksum file contains not only sha but also asset names.

```console
$ curl -L https://github.com/kubeshark/kubeshark/releases/download/38.5/kubeshark_38.5_checksums.txt
1507fca5aa4063ab31d0a8f8dc4916a4d03defa250e6ed190139c7cd7439b6fa  kubeshark_38.5_windows_arm64.tar.gz
15777e59512b5a85417c8ab7d7e4a8e65166c94773fab7466118be53d7f72fb3  kubeshark_38.5_windows_386.tar.gz
386b62c1fe042acf6d04e9c7be10c87e406cafbf72ef39f1cc6245eb1332fbb1  kubeshark_38.5_linux_amd64.tar.gz
63dfb7029466fb01f6858fda31357966bc0b2a4f7b1ad652204e92ac835327da  kubeshark_38.5_linux_386.tar.gz
7ed6e290eee3a9ca3776e41eee4033b34aa8e9fa3f66fa374df567e2d7efe44a  kubeshark_38.5_darwin_arm64.tar.gz
88193e5c1c73de2daa840e0a954753a719f15fb8c23d748d626c19e339f72dd6  kubeshark_38.5_linux_arm64.tar.gz
89bf5e7953f4c5774a27da6ebd4c07a3e37b56e5ce7feaca54b1c8000f7c5f23  kubeshark_38.5_windows_amd64.tar.gz
8b68f6de3152ce8a3952e9beafa4c05e83832321ec004966f7067899b0209790  kubeshark_38.5_darwin_amd64.tar.gz
```

Reply:
Oh, sorry. I misunderstood.

Reply:
We should use `kubeshark_38.5_checksums.txt` rather than `kubeshark_{{.OS}}_{{.Arch}}.sha256` because we can save API call.